modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/ShredModelEntry.scala

```diff
@@ -88,24 +88,24 @@ object ShredModelEntry {
     if (s == NullCharacter) "\\\\N"
     else s.replace('\t', ' ').replace('\n', ' ')
 
+  val extraCols = List(
```

Comment:
Here you wrap the column name in quotes (e.g. `""""schema_version""""`).  Then later in rdb-loader you remove the quotes again:

```scala
ShredModelEntry.extraCols.map(_._1.replaceAll(""""""", ""))
```

You might consider changing `extraCols` so that the column name is unquoted.  This means RDB loader can be cleaner because you don't need to unquote.  To compensate for this change, you could add the quoting somewhere inside `showProps`.

---

But there is more to consider!

By making this change, you have made `extraCols` part of the pulic-facing API of schema-ddl.  So at the very least it should have scaladoc, and an explicit type annotation.

And then also ask your question.... is the type `List[(String, String, String, String)]` a sensible type for a public-facing value?  Is there an alternative way of typing it, e.g. define a new case class?

Or alternatively, you could make `extraCols` private, and then define `extraColNames` as public:

```scala
val extraColNames: List[String] = extraCols.map(_._1)
```

Reply:
thanks Ian, just pushed a new commit on both this PR and rdb loader PR